doc/changelog_guidelines.md

```diff
@@ -0,0 +1,36 @@
+# CHANGELOG Guidelines
+
+The following guidelines should be followed when updating the CHANGELOG:
+- There should be an entry for every version, that includes the version number and release date.
+- Entries should be focused on communicating user-facing changes, considering that the main consumers of the CHANGELOG are the end users of SPIRE.
+- The types of changes should be grouped using the following categories:
+  - **Added**: New features that impact in the user experience.
+  - **Changed**: Changes in existing functionality.
+  - **Deprecated**: Features that will be removed in a future release.
+  - **Removed**: Features removed in this release.
+  - **Fixed**: Regular bug fixes. Should describe what the user would be experiencing if they were encountering the issue that is now fixed.
+  - **Security**: Security-related fixes.
+
+Categories that don't have an entry for the release are omitted in the CHANGELOG.
+
+The following is an example that includes all the categories:
+
+## [a.b.c] - YYYY-MM-DD
+
+### Added
+- Added AWS PCA configurable allowing operators to provide additional CA certificates for inclusion in the bundle (#1574)
```

Comment:
For the CHANGELOG sections whose header is a verb, e.g. "Added", "Changed", etc. I think we can omit leading verbs in each list item since it is kind of redundant.